File: Random/Examen/aux2022-2/pautaaux5_2022-2/Semantics/semantics.cpp
#include <iostream>
using namespace std;

int getNum5(int x) {
    return 5;
}

int getHii7() {
    cout<<"Hii\n";
    return 7;
}

int main() {
	cout<<getNum5(getHii7())<<"\n";
	return 0;
}

File: Random/Examen/aux2022-2/pautaaux5_2022-2/Semantics/semantics.java
class Main{
    
    static int getNum5(int x){
        return 5;
    }
    static int getHii7(){
        System.out.println("Hii");
        return 7;
    }
    public static void main(String[] args) {
        System.out.println(getNum5(getHii7()));
    }
}
